packages/@coorpacademy-components/src/molecule/setup-cohort/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import map from 'lodash/fp/map';
+import SetupCohortItem from '../setup-cohort-item';
+import Accordion from '../../organism/accordion/toggler';
+
+const SetupCohort = props => {
+  const {tabProps, slides = [], formatTitle} = props;
+  const slidesView = map.convert({cap: false})(
+    (slide, key) => <SetupCohortItem key={key} {...slide} />,
+    slides
+  );
+  return (
+    <div>
+      <Accordion
+        tabProps={map.convert({cap: false})(
+          (tab, idx) => ({
+            ...tab,
+            lastRow: idx + 1 === tabProps.length,
```

Comment:
it's strange.
I don`t want any special treatment for the last tab.

You must change the last tab('create new cohorte') by add a button to create a new cohorte. This button will create a new cohort (a new tab)

Reply:
I remove that